core/app/models/spree/shipping_category.rb

```diff
@@ -7,5 +7,18 @@ class ShippingCategory < Spree::Base
       has_many :shipping_method_categories
     end
     has_many :shipping_methods, through: :shipping_method_categories
+    
+    before_save :clear_cache
+
+    def self.default
+      Rails.cache.fetch("#{Rails.application.class.parent_name.underscore}_default_shipping_category") do
+        first
+      end
+    end
+
+    private
+    def clear_cache
+      Rails.cache.delete("#{Rails.application.class.parent_name.underscore}_default_shipping_category")
```

Comment:
The expression for the cache key is duplicated with: https://github.com/spree/spree/pull/6935/files#diff-d8e6742595fa0b47b62606ef77ddbe04R14

I suggest moving this one out to a separate method.